src/Microsoft.DotNet.Tools.Builder/CompileContext.cs

```diff
@@ -62,7 +60,7 @@ public bool Compile(bool incremental)
                 {
                     var dependencyProjectContext = ProjectContext.Create(dependency.Path, dependency.Framework);
 
-                    if (!NeedsRebuilding(dependencyProjectContext, new ProjectDependenciesFacade(dependencyProjectContext, _args.ConfigValue)))
```

Comment:
I'll make this change after the hella big one. I dont want to prolong it anymore. Will note this.